app_customers/android/app/build.gradle

```diff
@@ -25,6 +25,7 @@ if (flutterVersionName == null) {
 def dartEnvironmentVariables = [
     TRANSTU_APP_ENVIRONMENT: 'dev',
     TRANSTU_APP_NAME: 'TransTu.DEV',
+    APP_BASE_ADDRESS: '144.91.117.101:8301'
```

Comment:
this one should be empty or null. 
The IP address should never be hard coded. It should be loaded from github environment variable or secret.